src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AotTests.cs

```diff
@@ -418,17 +425,20 @@ public void HybridAOT ([Values ("armeabi-v7a;arm64-v8a", "armeabi-v7a", "arm64-v
 		}
 
 		[Test]
-		public void NoSymbolsArgShouldReduceAppSize ([Values (true, false)] bool enableHybridAot)
+		[Category ("LLVM")]
+		public void NoSymbolsArgShouldReduceAppSize ([Values ("", "Hybrid")] string androidAotMode)
```

Comment:
Should it be `"Normal"` instead of `""`?

Reply:
I think the test was previously leaving it blank, so I kept the behavior the same:
```diff
--			if (enableHybridAot)
--				proj.SetProperty ("AndroidAotMode", "Hybrid");
++			if (!string.IsNullOrEmpty (androidAotMode))
++				proj.SetProperty ("AndroidAotMode", androidAotMode);
```